framework/decode/vulkan_ascii_consumer_base.cpp

```diff
@@ -72,17 +71,29 @@ void VulkanAsciiConsumerBase::Process_vkAllocateCommandBuffers(
     uint32_t tabSize            = 4;
 
     auto createString = [&](std::stringstream& strStrm) {
-        FieldToString(strStrm, true, "return", toStringFlags, tabCount, tabSize, '"' + ToString(returnValue, toStringFlags, tabCount, tabSize) + '"');
-        FieldToString(strStrm, false, "device", toStringFlags, tabCount, tabSize, HandleIdToString(device));
+        FieldToString(strStrm, true, "type", toStringFlags, tabCount, tabSize, "\"vkCall\"");
```

Comment:
Can the writing of `type`, `index`, `name` all be factored out into `WriteApiCallToFile` across this change?